org.eclipse.xtext.testing/src/org/eclipse/xtext/testing/logging/LoggingTester.java

```diff
@@ -41,6 +41,18 @@ public class LoggingTester {
 
 	public static LogCapture captureLogging(Level level, Class<?> source, Runnable action) {
 		Logger logger = Logger.getLogger(source);
+		return captureLogging(logger, level, action);
+	}
+
+	/**
+	 * @since 2.35
+	 */
+	public static LogCapture captureLogging(Level level, String source, Runnable action) {
```

Comment:
The string represents the fully qualified name of the class? Maybe it's worthwhile to document the expected shape of the string in the Javadoc?